OneSignalSDK/onesignal/core/src/main/java/com/onesignal/common/consistency/enums/IamFetchRywTokenKey.kt

```diff
@@ -0,0 +1,12 @@
+package com.onesignal.common.consistency.enums
+
+import com.onesignal.common.consistency.models.IConsistencyKeyEnum
+
+/**
+ * Each enum is a key that we use to keep track of read-your-write tokens.
+ * Although the enums are named with "UPDATE", they serve as keys for tokens from both PATCH & POST
+ */
+enum class IamFetchRywTokenKey : IConsistencyKeyEnum {
+    USER_UPDATE,
+    SUBSCRIPTION_UPDATE,
```

Comment:
do we not need the create endpoints in here?

Reply:
[see comment](https://github.com/OneSignal/OneSignal-Android-SDK/pull/2168/files#diff-337b3906ee241eaf210ae8927ebb0767b98e8dc704ba9e2b549e81dcd494f174R7)

Reply:
ah ok. In that case would it make sense to just remove the `_UPDATE` suffix then?